packages/graphql/src/schema/resolvers/subscriptions/where/filters/filter-by-properties.ts

```diff
@@ -68,9 +68,15 @@ export function filterByProperties<T>({
                 }
             } else {
                 for (const [op, value] of Object.entries(v as Record<string, any>)) {
-                    const checkFilterPasses = getFilteringFn(op, operatorMapOverrides);
-                    if (!checkFilterPasses(receivedValue, value, fieldMeta)) {
-                        return false;
+                    if (op === "caseInsensitive") {
+                        if (!checkCaseInsensitiveFilters(value, receivedValue, fieldMeta)) {
+                            return false;
+                        }
```

Comment:
No change required, just a thought.

Instead of it being called `checkCaseInsensitiveFilters` you could have it as `hasCaseInsensitiveFilters` and return the result directly:

```suggestion
                    if (op === "caseInsensitive") {
                        return hasCaseInsensitiveFilters(value, receivedValue, fieldMeta)
```

Reply:
This is doing the filters checking for caseInsensitive, `hasCaseInsensitiveFilters` would be confusing, as this is actually doing the filtering, not just checking it has filters

Reply:
Alrighty!